test/l2/dai.ts

```diff
@@ -6,6 +6,7 @@ const { signERC2612Permit } = require('eth-permit')
 require('chai').use(require('chai-as-promised')).should()
 
 const MAX = '115792089237316195423570985008687907853269984665640564039457584007913129639935'
```

Comment:
MAX is already included in `helpers.js` as MAX_UINT256

Reply:
In fact we don't need `MAX_UINT256` as helper at all - I just learned that it is defined in ethers too: `ethers.constants.MaxUint256`.

Reply:
same for `ZERO_ADDRESS`, is defined in ethers too:
`ethers.constants.AddressZero`